src/main/java/com/spotify/github/v3/clients/GithubAppClient.java

```diff
@@ -69,8 +77,18 @@ public CompletableFuture<List<Installation>> getInstallations() {
    * @return a list of Installation
    */
   public CompletableFuture<Installation> getInstallation() {
+    return maybeRepo.map(repo-> github.request(
+        String.format(GET_INSTALLATION_REPO_URL, owner, repo), Installation.class, extraHeaders)).orElseGet(() -> getOrgInstallation(owner));
```

Comment:
```suggestion
        String.format(GET_INSTALLATION_REPO_URL, owner, repo), Installation.class, extraHeaders)).orElseGet(this::getOrgInstallation);
```
It can be used in this way, if we remove the owner param from `getOrgInstallation`